typescript/nodejs/README.md

```diff
@@ -0,0 +1,102 @@
+# TypeScript Node.js Demos
+
+## Overview
+
+This directory contains TypeScript sample programs for various Ice components.
+These examples are provided to get you started on using a particular Ice feature
+or coding technique.
+
+The following subdirectories contain sample programs:
+
+- [Glacier2](./Glacier2) is a simpler chat client that uses the Glacier2 service.
+
+- [Ice](./Ice) has several examples that illustrate fundamental Ice concepts.
+
+- [Manual](./Manual) contains complete examples for some of the code snippets
+in the [Ice manual][1].
+
+Refer to the [C++11 demos](../cpp11) for more examples that use Ice services
+such as IceGrid and IceStorm.
+
+## Build Instructions
+
+### Prerequisites
+
+The JavaScript sample programs require an installation of [Node.js][3].
+
+### Building the Demos
+
+#### Building the demos with the NPM packages
+
+With Node.js in your PATH, go to the [typescript/nodejs directory](../typescript/nodejs)
```

Comment:
Is this link correct?

Reply:
No that is bogus, it just the current directory of the README, maybe we should just remove it.

Reply:
👍